static/js/components/Dashboard.js

```diff
@@ -14,6 +14,9 @@ class Dashboard extends React.Component {
         </div>
         <div className="card-name">
           { SETTINGS.name }
+          <div className="card-student-id">
+            { profile.student_id }
```

Comment:
indeed! before I was passing the integer to the frontend and formatting it there.